src/components/searchDisplay.tsx

```diff
@@ -71,11 +100,19 @@ const renderSearchBox = (searchValue, updateSearchValue) => {
                 onChange={updateSearchValue}
                 style={{ color: 'rgba(0,0,0,.25)' }}
             />
-        </FormItem>);
+        </FormItem>
+    );
 };
 
-const renderSearchButton = (searchValue, runSearch) => {
-    if (searchValue && searchValue.length >= 1) {
+/* SEARCH BUTTON */
+
+interface SearchButtonProps {
+    searchValue: string;
+    runSearch: () => void;
+}
+
+const SearchButton: React.SFC<SearchButtonProps> = (searchValue, runSearch) => {
```

Comment:
@CallawayJay should just take a single parameter call `props`.  If you inspect the type, you will see it will be `SearchButtonProps`